app/js/openmrs.js

```diff
@@ -0,0 +1,119 @@
+angular.module('openmrs.resources', [ 'ngResource' ])
+	.factory('openmrsApi', function($resource) {
```

Comment:
Could you please follow https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y024 ?